smartsim/_core/control/controller.py

```diff
@@ -284,35 +287,40 @@ def _launch(self, manifest: Manifest) -> None:
         :param manifest: Manifest of deployables to launch
         :type manifest: Manifest
         """
-        orchestrator = manifest.db
-        if orchestrator:
-            if orchestrator.num_shards > 1 and isinstance(
-                self._launcher, LocalLauncher
-            ):
-                raise SmartSimError(
-                    "Local launcher does not support multi-host orchestrators"
-                )
-            if self.orchestrator_active:
-                msg = "Attempted to launch a second Orchestrator instance. "
-                msg += "Only 1 Orchestrator can be active at a time"
-                raise SmartSimError(msg)
-            self._launch_orchestrator(orchestrator)
+
+        orchestrators = manifest.db
+        if orchestrators:
+            # Loop over deployables to launch and launch multiple orchestrators
+            for orchestrator in orchestrators:
+                if orchestrator:
+                    if orchestrator.num_shards > 1 and isinstance(
+                        self._launcher, LocalLauncher
+                    ):
+                        raise SmartSimError(
+                            "Local launcher does not support multi-host orchestrators"
+                        )
+                    # if self.orchestrator_active:
+                    #     msg = "Attempted to launch a second Orchestrator instance. "
+                    #     msg += "Only 1 Orchestrator can be active at a time"
+                    #     raise SmartSimError(msg)
```

Comment:
Should this code be modified to see if we're attempting to launch a second orchestrator with the same db_id?

Reply:
Could do? But I am already adding a check before it even gets here? I suppose I can do the check here too? Or maybr it is better done here?